modules/Bio/EnsEMBL/G2P/Utils/Downloads.pm

```diff
@@ -74,43 +111,47 @@ sub download_data {
     }
   }
 
+  # For each genomic feature preload all genomic feature synonyms.
+  # The synonyms will be stored as prev symbols in the download file
+
   my $sth = $dbh->prepare('SELECT genomic_feature_id, name FROM genomic_feature_synonym;');
   $sth->execute() or die 'Could not execute statement: ' . $sth->errstr;
   while (my $row = $sth->fetchrow_arrayref()) {
     my ($id, $value) = @$row;
     $gfid2synonyms->{$id}->{$value} = 1;
   }
 
-  $sth = $dbh->prepare(q{SELECT genomic_feature_disease_id, created FROM genomic_feature_disease_log WHERE action='create';});
+  # Retrieve the date when the entry was created, when the GenomicFeatureDisease was added to the panel
+  # This information needs to be extracted from the genomic_feature_disease_panel_log table
+
+  $sth = $dbh->prepare(q{SELECT genomic_feature_disease_panel_id, created FROM genomic_feature_disease_panel_log WHERE action='create';});
   $sth->execute() or die 'Could not execute statement: ' . $sth->errstr;
   while (my $row = $sth->fetchrow_arrayref()) {
-    my ($gfd_id, $created) = @$row;
-    $gfd_create_dates->{$gfd_id} = $created;
+    my ($gfd_panel_id, $created) = @$row;
+    $gfd_panel_create_dates->{$gfd_panel_id} = $created;
   }
   $sth->finish();
 
+  # create where clause to decide which data to download
+  # default where clause: only download entries which are visible
   my $where = ($panel_name eq 'ALL') ? 'WHERE gfdp.is_visible = 1' : "WHERE a.value = '$panel_name' AND gfdp.is_visible = 1";
```

Comment:
On my test, here the $panel_name is empty and the downloaded file is also empty.

Reply:
Are you logged in? And which panel are you downloading?

Reply:
Are you also using this PR https://github.com/EBI-G2P/gene2phenotype_app/pull/22?

Reply:
I was missing that PR. Now it works!